abb_irb1600_support/config/joint_names_irb1600.yaml

```diff
@@ -0,0 +1 @@
+controller_joint_names: ['joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
```

Comment:
The name of this file should actually include the variant specific suffixes ('12'?).